generated.go

```diff
@@ -23,10 +23,21 @@ const Type = "qingstor"
 
 // Service available pairs.
 const (
+	// AdditionalUserAgent
+	pairAdditionalUserAgent = "qingstor_additional_user_agent"
```

Comment:
Maybe we can add `user_agent` in global?

Reply:
You mean in move this pair into go-storage?

Reply:
Yes, other service may also need this pair support?